src/pages/AdminViewPage/_AdminViewPage.scss

```diff
@@ -111,7 +111,6 @@
   .btn:hover {
     filter: brightness(80%);
   }
-
   position: relative;
```

Comment:
To bym przeniósł na górę, tam gdzie jest deklaracja klasy, będzie czytelniejsze

Reply:
zrobione